debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/RowMapperTest.java

```diff
@@ -173,4 +174,24 @@ public void testGetTableId() throws SQLException {
             assertThat(tableId).isNull();
         }
     }
+
+    @Test
+    @FixFor("DBZ-2555")
+    public void testGetTableIdWithVariedCase() throws SQLException {
+        Mockito.when(rs.getString(8)).thenReturn("Schema");
+        Mockito.when(rs.getString(7)).thenReturn("table");
+        TableId tableId = RowMapper.getTableId("CATALOG", rs);
+        assertThat(tableId.toString().equals("CATALOG.Schema.table")).isTrue();
+        verify(rs).getString(8);
+        Mockito.when(rs.getString(8)).thenThrow(SQLException.class);
+
+        tableId = null;
+        try {
+            tableId = RowMapper.getTableId("catalog", rs);
+            assertThat(1 == 2).isTrue();
```

Comment:
:thinking:  assertTrue(1 == 2) ? you want to make that fail, right?

Reply:
Pushed a commit to fix this.